corehq/ex-submodules/casexml/apps/phone/restore.py

```diff
@@ -636,7 +636,8 @@ def _get_asynchronous_payload(self):
             # context associated with this side of the fork will not be
             # recorded since it is async (see self.get_response).
             with self.timing_context("wait_for_task_to_start"):
-                task = get_async_restore_payload.delay(self)
+                # pass in extra args to have information about the task when viewing in celery/flower
+                task = get_async_restore_payload.delay(self, self.domain, self.restore_user.username, self.params)
```

Comment:
`self.params` is just another class so that won't be helpful. The ideal would be to serialize to JSON and then deserialize on the other side but that seems out of the scope of this PR